index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>MarkEllisDev Portfolio</title>
+
+	<meta charset="utf-8">
+	<link rel="stylesheet" type="text/css" href="css/styles.css">
+
+</head>
+<body>
+	<header>
+		<h1>Mark Ellis - Developer Portfolio</h1>
+	</header>
+
+	<article>
+		<h2>Bio</h2>
+		<img id="profilePic" src="images/mongoose_smoking.jpg" alt="Mark Ellis profile pic">
+		<p>This will eventually be my bio. Right now it's a placeholder.</p>
+
+	</article>
+
+	<article>
+		<table class=tableLayout3>
+			
+
+	 This will contain sites and links to sites I like, but for now, it contains my two older sites.
+
+	 		<td>
+	 			<h3><a href="http://emersonmedia.net/emerson_MEDIA/Welcome.html"> Emerson Media </a> </h3>
+	 		</td>
+
+	 		<td>
+	 			<h3><a href="http://markellisllc.com/"> Mark Ellis LLC </a>	</h3>
+	 		</td>
+
+		</table>
+	</article>
+
+	<article>
+	two future projects
+		<table>
+			<td width="50%">Future Project 1</td>
+			<td width="50%">Future Project 2</td>
+	</article>
+
+<!-- Footer element with email and professional social media links; Github, twitter, LinkedIn, etc. -->
+
+	<footer> 
```

Comment:
Be very careful about leaving trailing whitespace in your code. It will inevitably lead to merge conflicts. You can see whitespace by adding a plugin called Trailing Spaces to your sublime, or you can add this line to your User Preferences and stop them out right. "trim_trailing_white_space_on_save": true.